docqueries/driving_distance/test.conf

```diff
@@ -9,6 +9,7 @@
             doc-pgr_drivingDistance
             )],
         'documentation' => [qw(
+            dijksraDD-issue729
```

Comment:
I think this should be removed, because we are not using the query on the documentation (docs.pgrouting.org, aka in the rst files)

Reply:
okay

Reply:
i removed it further commits already 👍